app/src/main/res/values/strings.xml

```diff
@@ -459,19 +459,27 @@
     <string name="wallpaper_download_error_snackbar_action">Try again</string>
     <!-- Snackbar message for when wallpaper couldn't be selected because of the disk error -->
     <string name="wallpaper_select_error_snackbar_message">Couldn’t change wallpaper</string>
-
-    <!-- Label for switch which toggles the "tap-to-switch" behavior on home screen logo -->
-    <string name="wallpaper_tap_to_change_switch_label_1" moz:removedIn="105" tools:ignore="UnusedResources">Change wallpaper by tapping Firefox homepage logo</string>
-    <!-- This is the accessibility content description for the wallpapers functionality. Users are
-    able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
-    <string name="wallpaper_logo_content_description" moz:removedIn="105" tools:ignore="UnusedResources">Firefox logo - change the wallpaper, button</string>
+    <!-- Text displayed that links to website containing documentation about the "Seasonal wallpapers". -->
+    <string name="wallpaper_learn_more">Learn more</string>
+    <!-- Text for classic wallpapers title. -->
+    <string name="classic_wallpaper_title">Classic Firefox</string>
+    <!-- Text for seasonal wallpapers title. -->
+    <string name="seasonal_wallpaper_title">Limited Edition</string>
+    <!-- Text for seasonal wallpapers description. -->
+    <string name="seasonal_wallpaper_description">The new Independent Voices collection.</string>
```

Comment:
```suggestion
    <string name="seasonal_wallpaper_description">The new Independent Voices collection. %s</string>
```
This string actually needs to take a parameter, where %s is "Learn more". The reasoning for this is that when it comes to localization, you can't simply join 2 strings together and assume the order because this won't be the cause for RTL languages.

Reply:
I imagine you will also need a string without the %s for situations where there is no learn more URL.